source/usage-examples/bulkWrite.txt

```diff
@@ -0,0 +1,93 @@
+=======================
+Perform Bulk Operations
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+:node-api:`collection.bulkWrite <Collection.html#bulkWrite>` lets you
+perform bulk write operations against a *single* collection. With
+``collection.bulkWrite()``, you specify a list of operations to perform and
+MongoDB then executes those operations in bulk. Bulk write supports
+``insertOne``, ``updateOne``, ``updateMany``, ``deleteOne``,
+``deleteMany``, and ``replaceOne`` operations. Refer to the method
+documentation for full details.
+
+``bulkWrite()`` accepts the following parameters:
+
+- ``operations``: specifies the bulk operations to
+  perform. Each operation is passed to ``bulkWrite()`` as an object in
+  an array.
+
+- ``options``: *optional* settings that affect the execution
+  of the operation, such as :manual:`write concern
+  </reference/write-concern>` and order.
+
+  By default, MongoDB executes bulk operations in the specified order
+  (i.e. serially). When executing operations in order, if an error occurs
```

Comment:
I think serially might actually be too jargon-y here, but I can't deny it's correct and keeps things brief. If you can think of a simpler way to explain this, it might be easier for less-experienced users. Something like:
 "executes bulk operations in the order they are provided"?
"in order, so the first item in the provided collection completes before the second begins"?
but hopefully less awful and wordy than what I've said could be nice.

Reply:
Rephrased for newbies, though I kept 'i.e. serially' for folks who know what that means.